packages/aws-cdk-lib/aws-ec2/lib/instance.ts

```diff
@@ -315,6 +315,22 @@ export interface InstanceProps {
    * @default false
    */
   readonly ebsOptimized?: boolean;
+
+  /**
+   * If this parameter is set to true, the instance is enabled for AWS Nitro Enclaves; otherwise, it is not enabled for AWS Nitro Enclaves.
+   *
+   * @default false.
+   */
+  readonly nitroEnclaveEnabled?: boolean;
```

Comment:
Can we change the name to `enclaveEnabled`? I thought it is good because the name for the same one of L1 props is `enclaveOptions` without the word `nitro`.

Reply:
Thanks. I changed the property name.